d2b_migrate.info

```diff
@@ -1,4 +1,4 @@
-name = Drupal to Backdrop Migrate
+name = Drupal to Backdrop Migrate (D2B)
```

Comment:
Adding D2B here makes this module more findable on the permissions and modules page when searching for `d2b` in the filter box.